x-pack/plugins/fleet/server/services/epm/data_streams/get.ts

```diff
@@ -32,7 +33,17 @@ export async function getDataStreams(options: {
     : allDataStreams;
 
   const mappedDataStreams = filteredDataStreams.map((dataStream) => {
-    return { name: dataStream.name };
+    const dataStreamExtendedInfo = {
```

Comment:
Nit: This could be defined inline below as it's not always returned

Reply:
Addressed in [4d78f94](https://github.com/elastic/kibana/pull/170838/commits/4d78f94aff3bf7583273f87f7cc8d566754d390b).